sycl/test-e2e/AddressSanitizer/out-of-bounds/buffer/buffer.cpp

```diff
@@ -0,0 +1,34 @@
+// REQUIRES: linux, cpu
```

Comment:
I think merging all these tests into a single `.cpp` file might be beneficial for CI load, but we can address it separately for the entire E2E suite at once.